src/include/ikarus/finiteElements/mechanics/enhancedAssumedStrains.hh

```diff
@@ -268,6 +269,31 @@ namespace Ikarus {
         setEASType(0);
     }
 
+    void calculateDAndLMatrix(const auto& easFunction, const auto& par, Eigen::MatrixXd& DMat,
```

Comment:
you could write here a template member functions. It sohudl also be private.
```cpp
template<int enhancedStrainSize>
void calculateDAndLMatrix(const auto& easFunction, const auto& par, Eigen::Matrix<dozuble,enhancedStrainSize,enhancedStrainSize>& DMat,
```
otherwise this will be very slow in comparision